lib/kitchen/directions/bake_chapter_glossary/v1.rb

```diff
@@ -4,21 +4,85 @@ module Kitchen::Directions::BakeChapterGlossary
   class V1
     renderable
 
+    class SortableDefinition
+      attr_reader :definition_text
+
+      def initialize(definition_text:)
+        @definition_text = definition_text
+      end
+
+      def <=>(other)
+        I18n.sort_strings(definition_text, other.definition_text)
+      end
+    end
+
+    class Definition
+
+      # Text from tag dt
+      attr_reader :term_text
+
+      # Text from tag dd
+      attr_reader :description_text
+
+      # Clipboard with content of dl tag
+      attr_reader :def_content
+
+      def initialize(term_text:, description_text:, def_content:)
```

Comment:
since `def_content` is an element, let's call it `element` instead.  And actually, since the other two arguments are included in `def_content`, let's pull them out inside:

```ruby
def initialize(element)
  @term_text = I18nString.new(element.first('dt').text.downcase)
  @description_text = I18nString.new(element.first('dd').text.downcase)
  @element = element
end
```

Furthermore, `term_text` and `description_text` aren't really used except for sorting.  So I'd recommend the following:

```ruby
class Definition
  attr_reader :element

  def initialize(element)
    term = I18nString.new(element.first('dt').text.downcase)
    description = I18nString.new(element.first('dd').text.downcase)
    @sortable = [term, definition]
    @element = element
  end

  def <=>(other)
    sortable <=> other.sortable
  end
 
  protected

  attr_reader :sortable
end
```

Protected attributes are readable by other instances of the same class.  No other class needs this information.